packages/ui-extensions/docs/surfaces/point-of-sale/reference/components/Section.doc.ts

```diff
@@ -1,4 +1,8 @@
 import {ReferenceEntityTemplateSchema} from '@shopify/generate-docs';
+import {generateCodeBlock} from '../helpers/generateCodeBlock';
+
+const generateCodeBlockForComponent = (title: string, fileName: string) =>
```

Comment:
```suggestion
const generateCodeBlockForSection = (title: string, fileName: string) =>
```

This is a nit so up to your discretion for changes